src/anomalib/utils/metrics/thresholding/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Thresholding metrics"""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from .base import BaseAnomalyThreshold
+from .f1adaptive_threshold import F1AdaptiveThreshold
```

Comment:
Which one would look better, `f1adaptive_threshold` or `f1_adaptive_threshold`?

Reply:
I like the second one